### PR TITLE
Feature/align concatenate

### PR DESCRIPTION
**Pull request by patrick-tolosa**
#### What does this PR solve?
Feature/align concatenate to the specification

#### Final checklist
- [X] JSdoc - Add proper docstrings based on our PHP implementation
- [X] Implementation is aligned to Spec.
- [X] Tests - Add proper tests to the added code

### CODE REVIEW

File: src/transformation/Transformation.ts [hunk 1]
```diff
@@ -41,6 +41,11 @@ class Transformation {
     return this;
   }
 
+  addRawAction(raw: string) {
```
Comment (strausr) [inline src/transformation/Transformation.ts:44]: this is unrelated to concatenate right?

**Review by strausr — APPROVED** [review]

File: src/transformation/Transformation.ts (hunk 1 above)
Comment (patrick-tolosa) [reply src/transformation/Transformation.ts:44]: Somewhat related,
I had to include it to support this syntax:
`concatSourceTx.addRawAction(this.getTransitionString());`

We add a custom action to the source passed to concatenate.
